internal/cmd/eventgen/templates/eventfile.tmpl

```diff
@@ -19,9 +19,26 @@
 package eiffelevents
 
 import (
+	"time"
+
 	"github.com/clarketm/json"
+	"github.com/google/uuid"
 )
 
+// New{{.StructName}} creates a new struct pointer that represents
+// major version {{.MajorVersion}} of {{.EventType}}.
+// The returned struct has all required meta members populated.
+// The event version is set to the most recent {{.MajorVersion}}.x.x
+// currently known by this SDK.
+func New{{.StructName}}() (*{{.StructName}}, error) {
```

Comment:
Right now error is always nil, but do you expect that this function might fail in future editions?

Reply:
Yes, the modifier functions that I intend to add in a follow-up could potentially fail so I'm including the possibility of errors already here.